Makefile

```diff
@@ -33,3 +33,13 @@ image: build
 	docker build -t $(REGISTRY)/$(TARGET):${VERSION} .
 push: image
 	docker push $(REGISTRY)/$(TARGET):${VERSION}
+
+run-local: build
+	./$(TARGET) --cloud-config=<path to cloud.conf> \
+	    --v=<log_level> \
+		--cluster-name=<cluster_name> \
+		--use-service-account-credentials=true \
+		--bind-address=127.0.0.1 \
+		--cloud-provider=openstack \
+		--kubeconfig=<path-to-kubeconfig> \
+		--authentication-kubeconfig=<path-to-kubeconfig>
```

Comment:
Better to make this run with env variables. Or should we just make it a standalone script? It feels better suited to run as a standalone script rather than a makefile action.